home/migrations/0099_migrate_content_page_templates_to_standalone_templates.py

```diff
@@ -1,37 +1,105 @@
 from django.db import migrations
 from typing import Any
+from django.db.models import Count
+
+
+def rename_duplicate_whatsapp_titles(ContentPage: Any) -> None:
+    duplicate_whatsapp_titles = (
+        ContentPage.objects.values("whatsapp_title")
+        .annotate(count=Count("whatsapp_title"))
+        .filter(count__gt=1)
+        .values_list("whatsapp_title", flat=True)
```

Comment:
We only need to do this for `ContentPage`s with `is_whatsapp_template=True` right?